repository/src/main/java/org/alfresco/repo/domain/node/AbstractNodeDAOImpl.java

```diff
@@ -604,6 +605,17 @@ private TransactionEntity getCurrentTransaction()
         return txn;
     }
     
+    public Long getCurrentTransactionCommitTime()

```

Comment:
In order to avoid useless access to the database, I decide to use ThreadLocal in order to retrieve the current commit time, as did for the id, instead of reading it from the database.